lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -1,10 +1,36 @@
 class ActiveModel::Serializer::Adapter::JsonApi < ActiveModel::Serializer::Adapter
+        # JsonApiObject: namespace for functions that build
+        # JSON API objects
+        module JsonApiObject
```

Comment:
Yeah, just not sure if we'll go with this as a pattern. So, I'm incubating it here